src/modules/filelist/File.jsx

```diff
@@ -118,7 +117,7 @@ const File = ({
   // because they are magic folder created by the stack
   const canInteractWithFile =
     attributes._id !== 'io.cozy.files.shared-drives-dir' &&
-    attributes._id !== TRASH_DIR_ID
+    !attributes._id.endsWith('.trash-dir')
```

Comment:
why use a literal string instead of a constant here?

Reply:
This allows me to manage all the trash from both doctype `io.cozy.files` and `io.cozy.remote.nextcloud.files` in one line. You would have prefer something like that : 
```
const canInteractWithFile =
    attributes._id !== SHARED_DRIVES_DIR_ID &&
    attributes._id !== TRASH_DIR_ID &&
    attributes._id !== NEXTCLOUD_TRASH_DIR_ID
```
Maybe this constant should be provided by cozy-client or cozy-stack-client as they are already set as type [here](https://github.com/cozy/cozy-client/blob/556aef794fd9cdb577b7c22860f932187c355856/packages/cozy-client/types/types.d.ts#L8). 

Note:  I tend to put the doctype directly because I find them quicker to search globally in a project.